src/homepageExperience/utils.ts

```diff
@@ -100,3 +102,18 @@ export const HOMEPAGE_NAVIGATION_STEPS_ARDUINO = [
     glyph: IconFont.StarSmile,
   },
 ]
+
+// Each onboarding page in the wizard has a single h1 at the top of it. Attempt to scroll it into view smoothly
+// Set a timeout of 0 so that this function call gets run after react has a had a chance to update state and
+// re-render on the main thread.
+export const scrollNextPageIntoView = () => {
+  if (isFlagEnabled('firstMileScrollTop')) {
+    setTimeout(() => {
+      document.querySelector('h1').scrollIntoView({
```

Comment:
I was going to suggest a classname, but it looks like these `h1s` don't have 'em.

Reply:
probably less risky, but it would require changing every page in every first mile flow to add a class to the `h1`s